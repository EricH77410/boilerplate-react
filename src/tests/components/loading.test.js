import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../components/Loading';

test('should render loading correctly', ()=>{
    const wrapper = shallow(<Loading/>);
    expect(wrapper).toMatchSnapshot();
})