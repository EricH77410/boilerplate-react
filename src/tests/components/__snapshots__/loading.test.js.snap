// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__img"
    src="/images/loader.gif"
  />
</div>
`;
